.travis.yml

```diff
@@ -9,9 +9,10 @@ go:
 env:
   global:
     - GOTFLAGS="-race"
-  matrix:
     - BUILD_DEPTYPE=gomod
-
+  matrix:
+    - GOTFLAGS="-race -tags=openssl"
+    - GOTFLAGS="-tags=openssl"
```

Comment:
shouldn't we have a build in the matrix to test the go implementation as well?

Reply:
(oops)